src/mbgl/map/transform.cpp

```diff
@@ -46,6 +46,10 @@ Transform::Transform(MapObserver& observer_,
 #pragma mark - Map View
 
 bool Transform::resize(const Size size) {
+    if (!size) {
+        return false;
```

Comment:
I supposed that was the meaning of the return `bool` - should we make it `void` instead and throw on such cases?